Gemfile

```diff
@@ -28,13 +28,16 @@ group :test do
   gem "capybara"
 end
 
+group :production do
+  gem "honeybadger", "~> 4.0"
```

Comment:
Why do we want to do this? IMHO except local machine all errors should be logged once the code goes on cloud, even on non production envs such as UAT, Staging, development

Reply:
What's the difference between development and local machine?

The site only has dev (local) and production right now. So moving this gem to the production-only group make sense to me.

Reply:
local is basically developers machine, dev environment could be a hosted environment, staging/uat could be just before prod

dev environment is usually unstable, branch testing etc can happen there

staging is stable

I don't know what other environments are present before prod, but if there are any, ideally it's good if honey badger could catch them before going to prod